vaultwarden/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: vaultwarden
 description: Unofficial Bitwarden compatible server written in Rust
 type: application
-version: 0.3.0
+version: 0.4.0
```

Comment:
No need to bump the version, I'll do it when I publish an updated chart.

Reply:
cool